techmatters/flex-plugins
# Chi 1435 refactor contact saving logic redux promise

Primary reviewer: @stephenhand 

## Description

- **Feature refactor:** Refactor contact save logic into redux-promise middleware - part 1 (`updateContactsFormInHrm`).

The ticket handles the refactoring of network calls for contact to be handled in our Redux layer rather than the component layer. This is the first PR for the `updateContactsFormInHrm` service layer method.

### Checklist

### Related Issues
Fixes [CHI-1435](https://tech-matters.atlassian.net/browse/CHI-1435)

### Verification steps

- Search for a contact
- Select a contact you can edit
- Edit any aspect of the contact
- Verify that the edit was successful

[CHI-1435]: https://tech-matters.atlassian.net/browse/CHI-1435?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
